app/src/main/java/com/example/pygmyhippo/user/EventFragment.java

```diff
@@ -208,7 +208,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         // For admin to delete the event's QR code
         deleteQRCodeButton.setOnClickListener(buttonView -> {
             Log.d("EventFragment", "Delete QR Code Button pressed");
-            // TODO: Delete QR Code.
+            DBhandler.deleteQRHashData(event.getEventID(), this);
+            deleteQRCodeButton.setBackgroundColor(0xFF808080);
```

Comment:
are you trying to disable the qr code button? you can just set visibility to invisible if so

Reply:
what's the difference between this and that?

Reply:
would setVisability grey it out?

Reply:
setvisibaility will just make it invisible if that's what youre going for

Reply:
no, I just wanted the button to grey out and not do anything. a visual indication that the event already had a hash data removed

Reply:
there are sine videos of the changes in the pr if you want to see what ikm going for